### PR TITLE
OI-2636: New guidelines page

### PR DESCRIPTION
**Pull request by moz-rotimib**
Adds new guidelines page

### CODE REVIEW

File: web/src/components/pages/guidelines/guidelines.css [hunk 1]
```diff
@@ -0,0 +1,555 @@
...
+                                    font-size: var(--font-size-xs);
+                                }
+                            }
+
+                            p:nth-child(1) {
+                                margin-bottom: 6px;
+                                font-style: italic;
+                            }
+
+                            p:nth-child(2) {
```
Comment (moz-dfeller) [inline web/src/components/pages/guidelines/guidelines.css:338]: same here as mentioned above

File: web/src/components/pages/guidelines/guidelines.css [hunk 2]
```diff
@@ -0,0 +1,555 @@
...
+                                width: 220px;
+                            }
+
+                            p {
+                                @media (--sm-down) {
+                                    font-size: var(--font-size-xs);
+                                }
+                            }
+
+                            p:nth-child(1) {
```
Comment (moz-dfeller) [inline web/src/components/pages/guidelines/guidelines.css:333]: same here as mentioned above

File: web/locales/en/messages.ftl [hunk 3]
```diff
@@ -1588,3 +1588,95 @@ create-profile-button = Create a profile
...
+adding-sentences-subheader-special-characters-explanation-2 = Languages can have their own validation rules with additional requirements when they localize. If there is no specific validation file for a language, the English rules will show for contributors.
+adding-sentences-subheader-offensive-content = Offensive Content
+adding-sentences-subheader-offensive-content-explanation = If the sentence is offensive or upsetting - for example because of explicit content, or another violation of our <communityGuidelines>community participation guidelines</communityGuidelines> - you should reject the sentence. You can also reach out to us at <emailFragment>commonvoice@mozilla.com</emailFragment> to escalate the issue.
+reviewing-sentences-explanation-1 = If the sentence meets the criteria above, click the "Yes" button.
+reviewing-sentences-explanation-2 = If the sentence does not meet the above criteria, click the "No" button.
+reviewing-sentences-explanation-3 = If you are unsure about the sentence, you may also skip it and move on to the next one.
+reviewing-sentences-explanation-4 = If you run out of sentences to review, please help us collect more sentences!
```
Comment (moz-dfeller) [inline web/locales/en/messages.ftl:1682]: No newline at end of file :D

**Review by moz-dfeller — CHANGES_REQUESTED** [review]

File: web/src/components/pages/guidelines/guidelines.css [hunk 4]
```diff
@@ -330,12 +329,12 @@
...
 
-                            p:nth-child(1) {
+                            & p.tip-text {
                                 margin-bottom: 6px;
                                 font-style: italic;
                             }
 
-                            p:nth-child(2) {
+                            & p.tip-explanation {
                                 font-size: var(--font-size-xs);
```
Comment (moz-dfeller) [inline web/src/components/pages/guidelines/guidelines.css:338]: Lovely :)

**Review by moz-dfeller — APPROVED** [review]

**Comment by drzraf** [comment]
This doesn't seem to address https://github.com/common-voice/common-voice/issues/3796 ?
Custom pronunciations rules won't be doable after such a refactor. Could this be seriously considered?

File: web/src/components/pages/guidelines/sidebar-content/voice-collection/misreadings.tsx [hunk 5]
```diff
@@ -0,0 +1,87 @@
+import { Localized } from '@fluent/react';
+import classNames from 'classnames';
+import * as React from 'react';
+
+import { ChevronDown } from '../../../../ui/icons';
+import { TextButton } from '../../../../ui/ui';
+import { SidebarContentProps } from '../../types';
+
+import { ExampleContent, ExampleTip } from './example-content-voice';
```
Comment (drzraf) [inline web/src/components/pages/guidelines/sidebar-content/voice-collection/misreadings.tsx:9]: I couldn't figure out where will live the actual guidelines (per language) and whether we'll be able to amend them on a per language basis.

Comment (drzraf) [reply web/src/components/pages/guidelines/sidebar-content/voice-collection/misreadings.tsx:9]: ping

**Pull request merged**